### ISSUE TITLE
URLs with .m3u8 or .mpd in it may mistakenly be handled by this extension

### ISSUE THREAD

**Issue by gkatsev** [krzemienski/abr-manifest-viewer-chrome#2]
For Video.js, we have a test page that allows direct sharing by including the source url in the page url. For example https://videojs-http-streaming.netlify.app/?debug=false&autoplay=false&muted=false&minified=false&liveui=false&partial=false&url=https%3A%2F%2Fd2zihajmogu5jn.cloudfront.net%2Fbipbop-advanced%2Fbipbop_16x9_variant.m3u8&type=application%2Fx-mpegurl&keysystems=&buffer-water=false
It would be good to either be able to configure sites that never trigger the extension or maybe update https://github.com/krzemienski/abr-manifest-viewer-chrome/blob/a6de927c9bf9c5b510b3052344285b59519b0803/background.js#L4 to not include the `*` after the filetypes, though, a lot of sources do come with tokens.

**Comment by krzemienski** [comment]
@gkatsev yeah I’ve noticed this as well and have the same thoughts as far as making it configurable. Thats a bit more effort than I can put at the current moment though. 

Maybe a quick work around would be to check the contents of that given url it’s about to parse and ensure it has `#` and/or `<`

**Comment by gkatsev** [comment]
Just checking for `#` or `<` won't be enough. Maybe checking that it starts with `#EXTM3U` for HLS and includes `<MPD` for DASH would be good and minimize false positives.

**Comment by gkatsev** [comment]
And yeah, I get that adding config options is not a simple task.

The fact that the plugin puts a link to the actual page at the top is very helpful and I've been using that.

**Comment by krzemienski** [comment]
Yeah that’s what’s it’s for :) I’ll take a stab at something. Another thought is to maybe filter out qparams and only look at base url before choosing what to do.
